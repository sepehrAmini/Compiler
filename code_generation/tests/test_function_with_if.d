void calc(){
    if(true){
        Print("Im in the FUNCTION and statement is true");
    }
}

int main(){
    calc();
}

