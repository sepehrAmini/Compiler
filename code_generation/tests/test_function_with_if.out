Im in the FUNCTION and statement is true